entity-types/ext-air_conditioner/definition.yml

```diff
@@ -16,6 +17,10 @@ synthesis:
     tags.container_service:
       entityTagName: container_service
       multiValue: false
+  # Add a 4 hour ttl on all tags ingested in metric api using tags. prefix
+  prefixedTags:
```

Comment:
I think thep roblem in this one is that the is not under a `rules` list as the other are.

I don't know if we added support for this older format to define prefixedTags so if you can please convert this into:

```
synthesis:
  rules:
    - name: device_name
      identifier: ....